tests/test_parser.py

```diff
@@ -29,15 +27,28 @@ def bytes_to_service_info(payload: bytes) -> BluetoothServiceInfo:
     )
 
 
-def test_parsing_outdoor():
+def test_parsing_v5():
     device = RuuvitagBluetoothDeviceData()
-    advertisement = bytes_to_service_info(OUTDOOR_SENSOR_DATA)
+    advertisement = bytes_to_service_info(V5_SENSOR_DATA)
     assert device.supported(advertisement)
     up = device.update(advertisement)
-    expected_name = "RuuviTag EFAF"
+    expected_name = "RuuviTag DCFE"
```

Comment:
Nothing V5-related should change in this PR (except the name of the `OUTDOOR_SENSOR_DATA` variable).